test/unit/utils/multer.test.js

```diff
@@ -0,0 +1,64 @@
+const chai = require("chai");
+const { expect } = chai;
+const sinon = require("sinon");
+
+const cleanDb = require("../../utils/cleanDb");
+
+const multer = require("../../../utils/multer");
+
+describe("multer", function () {
+  afterEach(async function () {
+    await cleanDb();
+    sinon.restore();
+  });
+  describe("fileFilterImagesOnly", function () {
+    const callBack = (err, success) => {
+      return { err, success };
+    };
+    const fileType = [{ mimetype: "image/png" }, { mimetype: "test" }];
+    it("should pass without any error", async function () {
+      const response = await multer.fileFilterImagesOnly({}, fileType[0], callBack);
```

Comment:
Instead of callback, we can here too use sinon spy.

Reply:
You can, but I think this is fine, What improvements do we get if Sinon is used here?

Reply:
Ok. It will be more consistent, that's the only benefit.